### ISSUE TITLE
Emojis aren't rendered correctly in Algolia results

### ISSUE THREAD

**Issue by sebastiandedeyne** [spatie/murze.be#15]
For example, do a search on "Our packages have been downloaded one million times"

**Comment by LasseRafn** [comment]
Inspecting the search results, it seems like the actual import of posts haven't properly passed emojis, or maybe Algolia doesn't support emojis at all?

**Comment by jonsugar** [comment]
I'm playing about with your blog so I did a quick test. It seems that Algolia does support emojis. As can been seen in the screen shot.

![results](https://user-images.githubusercontent.com/1974648/33130292-c06b8bbe-cf8a-11e7-985a-ae13d61fb3b4.jpeg)

I tried making a new database with a non-unicode character set to see if there was a problem with character storage and translation but could not replicate the issue. It seems as though the laravel `database.php` config takes care of that with:

```
'charset' => 'utf8mb4',
'collation' => 'utf8mb4_unicode_ci',
```

which sets the correct character set for the tables during migration even if the database was created with a different character set.

I commented out the lines in `database.php` and created a new non-unicode database. I ran the migrations and database seed. I then tried to add an emoji and the database threw an error.

Finally, the response from murze.be. We can see that Algolia does not return the Emoji.

![results](https://user-images.githubusercontent.com/1974648/33131482-8f4409b8-cf8e-11e7-8de6-0ae47ce2a113.jpeg)

Without access to your index on Algolia it seems as if somehow the initial import of the data did not parse the emojis correctly even though they are clearly stored correctly in your database as they can be seen in the articles. 

I hope this helps 🤷‍♂️
